test/behaviour/reasoner/verification/Materialiser.java

```diff
@@ -109,7 +109,6 @@ void close() {
 
     private class Rule {
         private final com.vaticle.typedb.core.logic.Rule rule;
-        private final Concludable thenConcludable;
         private final Map<ConceptMap, Materialisation> conditionAnsMaterialisations;
         private boolean requiresReiteration;
 
```

Comment:
fyi our `typeDBRule` already has a name, since it lives in the `logic` package it should be `logicRule`